twoliter/embedded/Makefile.toml

```diff
@@ -939,25 +867,12 @@ dependencies = [
     "build-variant",
 ]
 
-[tasks.tuftool]
-script = [
-'''
-cargo install \
-  --locked \
-  --root tools \
-  --quiet \
-  --version ${PUBLISH_TUFTOOL_VERSION} \
-  tuftool
-'''
-]
-
 [tasks.create-infra]
-dependencies = ["infra-tools"]
 script = [
 '''
 set -e
 
-export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+export PATH="${TWOLITER_TOOLS_DIR}:${PATH}"
```

Comment:
Are we including `infrasys` also? I'd be OK with just dropping it here, not including it as a tool, and removing the sources later.

Reply:
> Are we including `infrasys` also? I'd be OK with just dropping it here, not including it as a tool, and removing the sources later.

I moved the sourcecode to here but I am not embedding it in twoliter. I will remove this makefile target.